docs/source/scaleVM.md

```diff
@@ -0,0 +1,165 @@
+# 🖥️ Scale Fedora & Windows VMs with Benchmark Runner
+
+This document provides automated scripts and configuration to **scale Fedora and Windows VMs** on OpenShift clusters. It verifies the VM login page and measures **bootstorm time**—the time it takes for VMs to become fully operational.
+
+---
+
+## 🚀 Features
+
+- Scale Fedora 37 VMs using **ephemeral storage**
+- Scale Windows 11 VMs using **ODF-backed storage** (via DataVolume)
+- Parallel VM loading across specified worker nodes
+- Optional metrics collection via **Elasticsearch**
+- Logs and artifacts are stored locally for post-run analysis
+
+---
+
+## 🧪 Prerequisites
+
+- OpenShift cluster with access to worker nodes
+- [`podman`](https://podman.io) installed
+- Access to the [benchmark-runner](https://quay.io/repository/ebattat/benchmark-runner) container image
+- Properly configured `KUBECONFIG` for cluster access
```

Comment:
That's implicit in "OpenShift cluster with access to worker nodes" unless there's something more specific that's needed, in which case it needs to be spelled out.

Reply:
Remove it